lighthouserc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    ci: {
+      collect: {
+        startServerCommand: 'npm run serve',
+        url: ['http://localhost:7000/en/']
+      },
+      upload: {
```

Comment:
We could have different uploads solution. See https://github.com/GoogleChrome/lighthouse-ci/blob/main/docs/configuration.md#upload

Here we're storing export on filesystem and saving it as artifact in CI.